kernel/tests/write.rs

```diff
@@ -1,41 +1,54 @@
+use std::collections::HashMap;
 use std::sync::Arc;
 
-use arrow::array::StringArray;
+use arrow::array::{Int32Array, StringArray};
 use arrow::record_batch::RecordBatch;
 use arrow_schema::Schema as ArrowSchema;
 use arrow_schema::{DataType as ArrowDataType, Field};
+use itertools::Itertools;
+use object_store::local::LocalFileSystem;
 use object_store::memory::InMemory;
 use object_store::path::Path;
 use object_store::ObjectStore;
+use serde_json::Deserializer;
 use serde_json::{json, to_vec};
 use url::Url;
 
 use delta_kernel::engine::arrow_data::ArrowEngineData;
 use delta_kernel::engine::default::executor::tokio::TokioBackgroundExecutor;
 use delta_kernel::engine::default::DefaultEngine;
 use delta_kernel::schema::{DataType, SchemaRef, StructField, StructType};
-use delta_kernel::{Error as KernelError, Table};
+use delta_kernel::Error as KernelError;
+use delta_kernel::{DeltaResult, Table};
 
-// setup default engine with in-memory object store.
+mod common;
+use common::test_read;
+
+// setup default engine with in-memory (=true) or local fs (=false) object store.
 fn setup(
     table_name: &str,
+    in_memory: bool,
```

Comment:
we never seem to pass `false`? do we need this?

Reply:
I'd originally had the code commented out that writes to file system but decided it would be better to actually implement it behind a flag. I used this to flip it to true and write out a table and then load that using spark to validate.